commands/azureCommands/create-registry.ts

```diff
@@ -0,0 +1,133 @@
+
+import * as vscode from "vscode";
+import { ContainerRegistryManagementClient } from 'azure-arm-containerregistry';
+import { ResourceManagementClient, SubscriptionModels } from 'azure-arm-resource';
+import { RegistryNameStatus } from "azure-arm-containerregistry/lib/models";
+import { ResourceGroup } from "azure-arm-resource/lib/resource/models";
+import { AzureCredentialsManager } from '../../utils/azureCredentialsManager';
+
+const teleCmdId: string = 'vscode-docker.createRegistry';
+
+export async function createRegistry() {
+    let subscription: SubscriptionModels.Subscription;
+    let resourceGroup: ResourceGroup;
+    try {
+        subscription = await acquireSubscription();
+        resourceGroup = await acquireResourceGroup(subscription);
+    } catch (error) {
+        return;
+    }
+    const client = AzureCredentialsManager.getInstance().getContainerRegistryManagementClient(subscription);
+
+    let registryName: string;
+    try {
+        registryName = await acquireRegistryName(client);
+    } catch (error) {
+        return;
+    }
+
+    const sku: string = await vscode.window.showInputBox({
+        ignoreFocusOut: false,
+        placeHolder: 'Basic',
+        value: 'Basic',
+        prompt: 'SKU? '
+    });
+
+    client.registries.beginCreate(resourceGroup.name, registryName, { 'sku': { 'name': sku }, 'location': resourceGroup.location }).then(function (response) {
```

Comment:
We should also acquire a location from the user instead of defaulting to that of the resource group.

Reply:
Just added that

Reply:
Fixed that to work with when you're creating a new resource group and/or new registry